README.md

```diff
@@ -1,26 +1,42 @@
-NodeSchool NYC :school: :statue_of_liberty:
-===
-[![Join us on Slack](http://nyjavascript-slack.herokuapp.com/badge.svg)](https://nyjavascript-slack.herokuapp.com/)
+# NodeSchool NYC
 
-What is NodeSchool?
-------
-NodeSchools are in-person hosted events in which open source workshops are used as curriculum and mentors are available in case you need help.
+Welcome to the NodeSchool San Francisco chapter's repository. We use this repository for our website and almost all our communication.
 
-* Main site: http://nodeschool.io
-* NYC Chapter site: http://nodeschool.io/nyc
-* Workshops: http://nodeschool.io/#workshopper-list
+You can see the website here: http://nodeschool.io/nyc
 
+## Talk to us
 
-## When is the next NodeSchool?
-**Feb 23rd, 6:30 PM**<br/>
-[RSVPs are OPEN on Meetup](https://www.meetup.com/NY-JavaScript/events/237477392/)
+We utilize [GitHub Issues](http://nodeschool.io/nyc/issues) like a message board and keep almost all of our communication in the open. If for any reason you need to contact an organizer privately you may contact us directly.
 
-## Can't make it to this one? There is one every month!
+## Team
 
-[See Past Events](https://github.com/nodeschool/nyc/blob/master/past-events.md) - 1 in 2014, 9 in 2015, 11 in 2016!
+- **Franziska Hinkelmann:**: [GitHub](https://github.com/fhinkel), [Twitter](https://twitter.com/fhinkel)
+- **Joe Sepi**: [GitHub](https://github.com/joesepi), [Twitter](https://twitter.com/joe_sepi/)
+- **Tierney Cyren**: [GitHub](https://github.com/bnb), [Twitter](https://twitter.com/bitandbang/)
+- **John Elliott**: [GitHub](https://github.com/johnelliott)
 
-[See Upcoming Events](https://github.com/nodeschool/nyc/blob/master/upcoming-events.md)
+## Contributing to the website
 
----
+The website is generated using Mustache templates, Stylus, and JavaScript. It runs on [GitHub Pages](https://pages.github.com/).
```

Comment:
Should this include a sentence that we accept PRs?

Reply:
Done!